cpuid.go

```diff
@@ -117,7 +117,7 @@ var flagNames = map[Flags]string{
 	SSE2:               "SSE2",               // P4 SSE2 functions
 	SSE3:               "SSE3",               // Prescott SSE3 functions
 	SSSE3:              "SSSE3",              // Conroe SSSE3 functions
-	SSE4:               "SSE4.1",             // Penryn SSE4.1 functions
+	SSE4:               "SSE4",               // Penryn SSE4.1 functions
```

Comment:
SSE4.1 + SSE4.2 = SSE4 as far as I understand.

This only detects SSE4.1

Reply:
I'm changing this back in v2.